### DISCUSSION TITLE
How does the ColorPicker control hide the button on the right?

### DISCUSSION THREAD

**Discussion by xinghaiyege** [AvaloniaUI/Avalonia#13566] [Q&A] [answered]
![image](https://github.com/AvaloniaUI/Avalonia/assets/93193390/29513f1c-9be4-453a-aff5-481ac6f865aa)

Avalonia version：11.0.5
Theme：Fluent

**Comment by timunie** [comment, marked as answer]
```xml
<Style Selector="ColorPicker /template/ DropDownButton /template/ PathIcon#DropDownGlyph">
  <Setter Property="IsVisible" Value="False" />
</Style>
<Style Selector="ColorPicker /template/ DropDownButton">
  <Setter Property="Padding" Value="0" />
</Style>
```

like that. Lookup original styles when in doubt.

**Reply by robloo** [reply]
Also note that if you are doing this I would potentially just re-template the ColorPicker to use a `Button` rather than a `DropDownButton`. That seems more correct but certainly isn't as easy as a solution.

https://github.com/AvaloniaUI/Avalonia/blob/9cd8dda5206ffbcb6705ada2ae8d3753aaaf9caf/src/Avalonia.Controls.ColorPicker/Themes/Fluent/ColorPicker.xaml#L38-L51
